squad/frontend/views.py

```diff
@@ -99,8 +100,19 @@ def group_home(request, group_slug):
     elif order_by == 'last_updated':
         projects_queryset = projects_queryset.order_by('-datetime')
 
-    display_all_projects = request.GET.get('all_projects') is not None
     num_projects = group.get_setting('DEFAULT_PROJECT_COUNT')
+
+    max_project_age_days = group.get_setting('MAX_PROJECT_AGE_DAYS')
```

Comment:
I thought it over, and what do you think about the name `SHOW_PROJECTS_ACTIVE_N_DAYS_AGO`?